packages/apps/project/src/components/templates/ActivitiesListView/ActivitiesListView.tsx

```diff
@@ -0,0 +1,241 @@
+          handlePreviousActivity();
+        } else {
+          setDataList(prevData => [...prevData, ...formattedData]);
+        }
+      } catch (error) {
+        console.error(error);
+      }
+    },
+    [project?.id, dateLimit, handlePreviousActivity],
+  );
+
+  useEffect(() => {
+    if (project?.id) {
+      fetchActivityData(startDate);
+    }
+  }, [fetchActivityData, startDate, project?.id]);
+
+  const formatDate = date => {
+    const day = String(date.getDate()).padStart(2, '0');
+    const month = String(date.getMonth() + 1).padStart(2, '0');
+    const year = date.getFullYear();
+    return `${month}/${day}/${year}`;
+  };
+
+  const convertToDate = dateStr => {
+    const [month, day, year] = dateStr.split('/').map(Number);
+
+    return new Date(year, month - 1, day);
+  };
+
+  const calculateTimeDifference = time => {
+    const date = new Date(time);
+    const now = new Date();
+
+    if (isNaN(date.getTime())) {
+      return;
+    }
+
+    const dateInMs = Date.UTC(
+      date.getFullYear(),
+      date.getMonth(),
+      date.getDate(),
+    );
+    const nowInMs = Date.UTC(now.getFullYear(), now.getMonth(), now.getDate());
+
+    const differenceInMilliseconds = nowInMs - dateInMs;
+
+    const differenceInDays = Math.floor(
+      differenceInMilliseconds / (1000 * 60 * 60 * 24),
+    );
+
+    if (differenceInDays > 0) {
+      return `${I18n.t('Project_UpdatedTimes', {
+        times: `${differenceInDays} ${I18n.t('Project_Day').toLowerCase()}${
+          differenceInDays > 1 ? 's' : ''
+        }`,
+      })}`;
+    } else {
+      const _differenceInMilliseconds = now.getTime() - date.getTime();
+      const differenceInHours = Math.floor(
+        _differenceInMilliseconds / (1000 * 60 * 60),
+      );
+      if (differenceInHours > 0) {
+        return `${I18n.t('Project_UpdatedTimes', {
+          times: `${differenceInHours} ${I18n.t('Project_Hour').toLowerCase()}${
+            differenceInHours > 1 ? 's' : ''
+          }`,
+        })}`;
+      } else {
+        const differenceInMinutes = Math.floor(
+          _differenceInMilliseconds / (1000 * 60),
+        );
+        return `${I18n.t('Project_UpdatedTimes', {
+          times: `${differenceInMinutes} ${I18n.t(
+            'Project_min',
+          ).toLowerCase()}${differenceInMinutes > 1 ? 's' : ''}`,
+        })}`;
+      }
+    }
+  };
+
+  const renderItem = ({item}) => {
+    const key = Object.keys(item)?.[0];
+
+    const activity = item?.[key]?.[0].activity || {};
+
+    const modelName = item?.[key]?.[0]?.modelName;
+
+    const tracks = activity?.tracks || [];
+
+    const time = item?.[key]?.[0].time || '';
+
+    const user = item?.[key]?.[0].user || '';
+
+    const utilityClass = item?.[key]?.[0].utilityClass;
+
+    return (
+      <View style={styles.item}>
+        <Text style={styles.updatedText}>{`${I18n.t(
+          'Project_UpdatedBy',
+        )} ${user} ${calculateTimeDifference(time)}`}</Text>
```

Comment:
Instead of displaying the time difference, maybe we can only display the time when it was modified as we already have the day information in the header